src/util/persistantStore.ts

```diff
@@ -10,4 +10,12 @@ export default {
   remove(entryName: string): void {
     return window.localStorage.removeItem(entryName);
   },
+
+  downloadSettings() {
+    return window.localStorage.getItem('library-series-list');
+  },
+
+  restoreSettings(settings: any) {
+    return window.localStorage.setItem('library-series-list', settings);
+  },
```

Comment:
I wouldn't make a separate function for these; you can have your other methods use the existing `persistantStore.write` and `persistantStore.read`, and as the `entryName` you can use the constant from constants/storeKeys.json:

https://github.com/xgi/houdoku/blob/f969754d37e07234a8902a7a6dbe34aa3e8b7950/src/constants/storeKeys.json#L3

Reply:
Check the changes and lemme know, also no styling was added, Even tho I know about React,I haven't touched Ant framework. You can probably do that in next release.